### DISCUSSION TITLE
crashes on animated gif

### DISCUSSION THREAD

**Discussion by fenio** [community/community#126924] [Mobile]
### Select Topic Area

Bug

### Body

When I'm trying to open animated GIF app simply crashes.
iOS 17.5.1
Github Mobile Version: 1.163.0 (131249690)

I'm attaching problematic file but it can be also tested by exploring my repo and clicking whatever demo GIF at  the demos section: https://github.com/fenio/pv-mounter

![mounted](https://github.com/community/community/assets/164051/9ad1cfb7-c43e-4e51-98e7-91cb897d03b5)

**Comment by esahmed21** [comment]
Thank you @fenio for reporting this issue! I was able to reproduce it using the repository you shared!

I will create internal bug for our team to get it fixed!

Thanks again for reporting and sharing steps to reproduce!
